hack/update-golden.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+go test $(go list -f '{{ .ImportPath }} {{ .TestImports }}' ./... | grep golden | awk '{print $1}' | tr '\n' ' ') -test.update-golden=true
```

Comment:
can you add a comment on what it does? this look like hebrewcr to me! (

Reply:
lol ok :sweat_smile: